benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/MeasureComposable.kt

```diff
@@ -91,7 +93,7 @@ suspend fun measureComposable(
             val start = markNow()
 
             repeat(frameCount) {
-                val frameStart = start + nextVSync
+                val frameStart = markNow()
```

Comment:
We mark frameStart from nextVSync intentionally to catch possible GC. Consider revert this.

Reply:
I accidentally added it in this PR. Let's move the discussion to https://github.com/JetBrains/compose-multiplatform/pull/5215/files#r1932225444

Will revert here

Reply:
Reverted